processes/third_party_vuln_process.md

```diff
@@ -77,7 +77,12 @@ At this point, a CVE should be requested through the HackerOne platform through
 [email](cve-assign@hackerone.com) that should include the Report ID and a summary.
 
 The vulnerability is considered as published when a Pull Request adding it
-to this repository is opened.
+to this repository is opened. Once the Pull Request is merged, and a new version
```

Comment:
usually when we disclose a vulnerability it already has a fixed version out. on the flip side, sometimes we disclose after the threshold (45 days) due to no response from the maintainer so in that case a deprecation will probably not  be relevant too.